src/LayoutManager/ActiveSelectionLayoutManager.spec.ts

```diff
@@ -39,17 +39,30 @@ describe('ActiveSelectionLayoutManager', () => {
         expect(objectOff).not.toHaveBeenCalled();
       });
 
-      it('a subscribed activeSelection should trigger layout on the object parent', () => {
+      it('a subscribed activeSelection should trigger layout on the object parent once per parent', () => {
```

Comment:
How does this test really test that?
Seems incomplete to me, am I missing something?

Reply:
the _subscription map is a 'new Map()` and so supports one value per key.
Once we test the key is per group and not per object, we know there is only one registration for that group.
It comes intrinsic from the type of key you use.

Reply:
Yes, but the test doesn't assert there aren't floating subscriptions
Or maybe it does actually, in the subscribe tests